components/event-signup.js

```diff
@@ -0,0 +1,213 @@
+"use client";
+
+import { useForm } from "react-hook-form";
+import { useState, useEffect } from "react";
+
+function EventSignup(eventProps) {
+  const [eventData, setEventData] = useState();
+  const [registerSuccess, setRegisterSuccess] = useState();
+  const [ticketUrl, setTicketUrl] = useState("");
+
+  useEffect(() => {
+    checkSlug();
+  }, []);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitSuccessful, isSubmitting },
+  } = useForm({
+    mode: "onTouched",
+  });
+
+  const checkSlug = async () => {
+    const eventData = fetch("/api/tito-check-slug")
```

Comment:
This is that API change. You will have event in the eventProps. So the fetch should be able to include the titoSlug in the uri

Reply:
👍

Reply:
Resolved in commit: ee1641a8fbf0b7d95a0cc3cc58f9add5dac2ef7e